decorators/README.md

```diff
@@ -1,3 +1,93 @@
 # Decorators
+> Testing recipes for decorators
 
-## In construction
+## Table of contents
+
+- [Boilerplate](#boilerplate)
+- [Boilerplate](#boilerplate---decorator-file)
+- [Boilerplate](#boilerplate---test-for-decorator-file)
+
+
+## Boilerplate
+
+Before start we need to initialize the related controller and mock it's dependencies.
+
+### Boilerplate - Decorator file
```

Comment:
The same of the controller and the filter explanations. We don't need to show the code here. You can talk only about the testing boilerplate